web-client/src/app.tsx

```diff
@@ -262,6 +263,11 @@ const app = {
       returnSequencePromise: true,
     });
 
+    // Expose Cerebral for testing
+    if (process.env.NODE_ENV !== 'production') {
+      (window as unknown as ITestableWindow).cerebral = cerebralApp;
+    }
```

Comment:
Exposing this so that manual tests (and, if I can get them to work in the future, automated tests) can be done more easily.

Reply:
Well this makes me a little nervous, but if we're not doing it in prod I reckon it's fine